.changeset/dark-radios-punch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@prairielearn/cache': minor
```

Comment:
Are you sure this isn't a breaking change?

Reply:
Ah.

Reply:
Reverted.